### ISSUE TITLE
Blank screen 

### ISSUE THREAD

**Issue by tweaver95** [ZenoArrows/The-Simpsons-Hit-and-Run#33]
I did as stated in the instructions.when I click on the game it's nothing but a blank screen.

**Comment by tweaver95** [comment]
Is anyone able to make a good tutorial on how to install it?

**Comment by dem1980** [comment]
Do you play with PAL game and PAL nro and the v4 works fine ?

**Comment by tweaver95** [comment]
I do not play with PAL

**Comment by tweaver95** [comment]
I do not play with PAL. I'm not sure if I installed it correctly or not though

**Comment by dem1980** [comment]
Do you have tested with 0.4 files before 0.5 ?

**Comment by tweaver95** [comment]
Yes I have. And same issue. I'm not sure if I'm using the right pc files/ putting everything in the right folders.
